addons/account/views/account_tax_views.xml

```diff
@@ -157,10 +158,13 @@
                             <div attrs="{'invisible': [('amount_type', '=', 'group')]}">
                                 <field name="country_code" invisible="1"/>
                                 <group string="Distribution for Invoices">
-                                    <field name="invoice_repartition_line_ids" nolabel="1" colspan="2"/>
+                                    <field name="invoice_repartition_line_ids" nolabel="1" colspan="2" context="{'default_document_type': 'invoice'}"/>
                                 </group>
                                 <group string="Distribution for Refunds">
-                                    <field name="refund_repartition_line_ids" nolabel="1" colspan="2"/>
+                                    <field name="refund_repartition_line_ids" nolabel="1" colspan="2" context="{'default_document_type': 'refund'}"/>
+                                </group>
+                                <group invisible="1">
+                                    <field name="repartition_line_ids"/>
```

Comment:
do we still need changes in this view?

Reply:
Hello @aliyatastemirova ,
Yes we require this change in the view. Without it 'invoice_repartition_line_ids' and 'refund_repartition_line_ids' won't be visible in the view. I request you to please check it without this field you would see considerable difference when you save the tax. Also if you have any suggestions please do mention them below.
Thank you

Reply:
Hi, that's probably because you remove `'invoice_repartition_line_ids', 'refund_repartition_line_ids'` from constrains above for `_validate_repartition_lines`. I think if you just add `repartition_line_ids` to them instead of replacing it should be okay.
We also used to create default repartition lines before this version, do you think we could fix that too within this PR? :slightly_smiling_face: :pray:

Reply:
Ǹo, only minimal changes are required
```diff
diff --git a/addons/account/views/account_tax_views.xml b/addons/account/views/account_tax_views.xml
index 867c509835b..12e864b1b56 100644
--- a/addons/account/views/account_tax_views.xml
+++ b/addons/account/views/account_tax_views.xml
@@ -40,6 +40,7 @@
                     <field name="sequence" widget="handle"/>
                     <field name="factor_percent" attrs="{'invisible': [('repartition_type', '=', 'base')]}"/>
                     <field name="repartition_type"/>
+                    <field name="document_type" invisible="1"/>
                     <field name="account_id" attrs="{'invisible': [('repartition_type', '=', 'base')]}" options="{'no_create': True}"/>
                     <field name="tag_ids"
                            widget="many2many_tags"
@@ -156,6 +157,7 @@
                         <page string="Definition" name="definition">
                             <div attrs="{'invisible': [('amount_type', '=', 'group')]}">
                                 <field name="country_code" invisible="1"/>
+                                <field name="repartition_line_ids" invisible="1"/>
                                 <group string="Distribution for Invoices">
                                     <field name="invoice_repartition_line_ids" nolabel="1" colspan="2"/>
                                 </group>
```

Adding the `repartition_line_ids` is not even needed actually; I wouldn't do it